composer.json

```diff
@@ -15,8 +15,8 @@
         "bjeavons/zxcvbn-php": "^0.4"
     },
     "require-dev" : {
-        "squizlabs/php_codesniffer" : "3.5.1",
-        "phpunit/phpunit" : "6.5.*",
+        "squizlabs/php_codesniffer" : "2.9.2",
```

Comment:
why is PHPCS being downgraded?

Reply:
Probably a rebase issue. I think this was issued before the PHPCS update. I'll change it.